lib/main.js

```diff
@@ -18,7 +20,11 @@ Object.assign(exports, {
     PlacesProvider.links.init();
     SearchProvider.search.init();
     options.telemetry = false;
-    app = new ActivityStreams(options);
+
+    Task.spawn(function*() {
+      options.clientID = yield ClientID.getClientID();
```

Comment:
I'll confirm if that's the client ID we want. This will yield us the telemetry client id. IIRC we wanted an FxA client ID/

Also, please remove mention of `storage.clientUUID` in lib/main.js, it won't be needed if we use another client id.